src/flamegraph/color/palettes.rs

```diff
@@ -269,11 +267,11 @@ mod tests {
             },
             TestData {
                 input: String::from("jdk/::[ki]"),
-                output: BasicPalette::Green,
+                output: BasicPalette::Yellow,
```

Comment:
Interesting.. Shouldn't this still be green?

Reply:
This case is contraversal, I think It should be c++ call due to `::`, so yellow looks correct.

Reply:
Hmm, curious.. My worry is that this is a test that stemmed from some real-world use-case where that popped up in Java code (because of `jdk`). But I could be misremembering its origins, and maybe it was just something we made up?

Reply:
I believe it is a synthetic test case to cover the priority matching of previous implementation(https://github.com/jonhoo/inferno/pull/131). In the current implementation the priority was changed. It does not look that a method in java could contain `::` character, but I have not seen `/` in c++ parts too.

Sure we can make an exception for this case in matching to keep this test case to work in the same way, but I believe the test result should be changed instead.

Reply:
Seems reasonable to me, I just wanted to double-check!